src/components/Footer/Footer.styles.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+import { ReactComponent as LogoSVG } from '../../assets/images/sign.svg';
```

Comment:
This is imported as base64 encoded string I guess. So I'd move the `sign.svg` inside the `src/components/Footer` directory. The logo is also not reused anywhere. But it's your call

Reply:
Agreed and changed

Reply:
I was actually saying that I would move sign.svg from `src/assets/images/sign.svg` to `src/components/Footer/sign.svg` and use it the way you did before

```
import { ReactComponent as LogoSVG } from './sign.svg'; 

export const Logo = styled(LogoSVG)`...`
```

But this is fine for me as well. I'll leave it up to you to change this or not

Reply:
Oh ok, I moved the images and deleted assets folder for now